### DISCUSSION TITLE
Android app on Zebra

### DISCUSSION THREAD

**Discussion by SergeoLacruz** [inventree/InvenTree#7477] [Q&A]
Hi,
Has anyone experience with porting the mobile app to something like this
https://www.zebra.com/de/de/products/mobile-computers/handheld/ps20.html
or a similar device. Just a pure barcode scanner, even wireless, is not always
useful as it has no UI. 
Michael

**Comment by SchrodingersGat** [comment]
No personal experience but I know that some users have had success with mobile barcode scanners like this.

We added "keyboard input mode" to accommodate these devices: 

https://docs.inventree.org/en/stable/app/barcode/#keyboard-input
